docs/getting_started/requirements.md

```diff
@@ -23,21 +25,21 @@ Using the latest listed version of each product or component is always recommend
 
 ## Web server
 
-- Nginx 1.18 or higher
+- Nginx 1.18+
 - Apache 2.4 (with required modules `mod_rewrite`, `mod_env` and recommended: `mod_setenvif`, `mod_expires`;
 event MPM is recommended, if you need to use prefork you also need the `mod_php` module)
 
 ## DBMS
 
-- MariaDB 10.3, 10.4, 10.5
+- MariaDB 10.3+
 - MySQL 8.0
 - PostgreSQL 10+
 
 ## PHP
 
 - 8.1
 - 8.0
-- 7.4
+- 7.4 (PHP 7.4 has reached its End of Life. Unless you have extended support from vendors like Debian or Zend, you should use PHP 8.)
```

Comment:
```suggestion
- 7.4 (PHP 7.4 has reached its End of Life. Unless you have extended support from vendors like Debian or Zend, you should use PHP 8.1)
```

Reply:
8.2 was just released, but I guess we're not supporting it just yet :) https://www.php.net/releases/8.2/en.php

Reply:
I don't expect any issues but we should do "official" certification as usually // cc @micszo

Reply:
We already have plans in the team. 🙂